packages/web3-eth-ens/src/config.js

```diff
@@ -4,7 +4,8 @@ var config = {
     addresses: {
         main: "0x314159265dD8dbb310642f98f50C066173C1259b",
         ropsten: "0x112234455c3a32fd11230c42e7bccd4a84e02010",
-        rinkeby: "0xe7410170f87102df0055eb195163a03b7f2bff4a"
+        rinkeby: "0xe7410170f87102df0055eb195163a03b7f2bff4a",
+        goerli: "0x112234455C3a32FD11230C42E7Bccd4A84e02010"
```

Comment:
Just leaving a link to where these addresses are listed (in case someone's ever curious):

https://docs.ens.domains/ens-deployments

Reply:
Oh, good idea! We could also add it as a file header of the ``config.js`` file as we have it in other places. WDYT?

Reply:
I've added the missing file header, variable type notation, and the source of those addresses with https://github.com/ethereum/web3.js/pull/3338/commits/c948aadd6cbe54f85aa42db2030726adc69ace3e. I will revert it if this is out of scope.

Reply:
Oh cool! Great.